docs/config.md

```diff
@@ -365,6 +366,8 @@ This directs the solver to use the `preciceAdapterFunctionObject` function objec
 which is part of the `libpreciceAdapterFunctionObject.so` shared library.
 The name `preCICE_Adapter` can be arbitrary. It is important that the library is loaded outside the `functions` dictionary when you want to use the custom boundary conditions that we provide with the FF module.
 
+The `errors strict` option is optional and [available since OpenFOAM v2012](https://www.openfoam.com/news/main-news/openfoam-v20-12/post-processing#post-processing-function-object-error-handling). Since the adapter is necessary to do a coupled simulation, this option instructs OpenFOAM to stop in case it faces issues with loading the adapter. For OpenFOAM versions that don't support this, remove the option.
```

Comment:
So, if not set, my OpenFOAM solver will just keep going, yes?

Reply:
Correct, same as when OpenFOAM cannot find the adapter.
Do you think this is particularly risky?
Should we take any further actions beyond mentioning this prominently in the changelog?